.github/workflows/testing.yml

```diff
@@ -195,3 +197,179 @@ jobs:
 
       - name: Check workability of benchmark tests
         run: make bench-deps bench DURATION=1x COUNT=1
+
+  testing_mac_os:
+    # We want to run on external PRs, but not on our own internal
+    # PRs as they'll be run by the push to the branch.
+    #
+    # The main trick is described here:
+    # https://github.com/Dart-Code/Dart-Code/pull/2375
+    if: (github.event_name == 'push') ||
+      (github.event_name == 'pull_request' &&
+        github.event.pull_request.head.repo.full_name != github.repository) ||
+      (github.event_name == 'workflow_dispatch')
+
+    strategy:
+      fail-fast: false
+      matrix:
+        golang:
+          - 1.13
+        runs-on:
+          - macos-10.15
+          - macos-11.0
+        tarantool:
+          - brew
+          - 1.10.10
+          - 2.8.1
```

Comment:
Is there no way to use pre-build packages on MacOS?

Reply:
I'm thinking on it in background. Of course, it is possible and there are different ways with cons and pros. We should setup some infrastructure and support it in setup-tarantool. Reach me if you want to participate and/or push this activity.

Reply:
Yeah, building Tarantool from sources for module CI seems painful. I think we must get rid of it if possible.

Reply:
Maybe cache will resolve the problem.  You can check it [here](https://github.com/tarantool/go-tarantool/actions/runs/3584174473/jobs/6030512155) macOS 12 Tarantool 2.8.1 test.